exchanges/bybit/spot_websocket.go

```diff
@@ -0,0 +1,50 @@
+package bybit
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/thrasher-corp/gocryptotrader/encoding/json"
+	"github.com/thrasher-corp/gocryptotrader/exchange/websocket"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/subscription"
+)
+
+func (e *Exchange) handleSpotSubscription(ctx context.Context, conn websocket.Connection, operation string, channelsToSubscribe subscription.List) error {
+	payloads, err := e.handleSubscriptions(conn, operation, channelsToSubscribe)
+	if err != nil {
+		return err
+	}
+	for _, payload := range payloads {
+		response, err := conn.SendMessageReturnResponse(ctx, request.Unset, payload.RequestID, payload)
+		if err != nil {
+			return err
+		}
+		var resp SubscriptionResponse
+		if err := json.Unmarshal(response, &resp); err != nil {
```

Comment:
You can remove this immediate assignment since there is a big scoped err variable above.

Reply:
prefer if short, won't resolve so people can yell at me.